docs/core/integration-quality-scale/rules/iqs027.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "IQS027 - Dependency is async"
+---
+
+IQS027 is as follows:
+
+Dependency is async
+
+## Reasoning
+
+Home Assistant works with asyncio to be efficient.
+If your library is not async, it needs to utilize executor jobs to not block the event loop.
+Ideally, your library should be async.
```

Comment:
Honestly, this line is highly conflicting with the title of the rule.

For me, this rule is pretty easy, async or gtfo :D

Reply:
I think it's meant as an explanation and not an instruction, but it can be misinterpreted.